docs/concept-signals.md

```diff
@@ -4,4 +4,32 @@ title: Signals
 sidebar_label: Signals
 ---
 
-TODO
+## Event Handling
```

Comment:
This is content that was in concept-event page  - just moved it here. Redirect was added.